src/engine/DynamicEntry.ts

```diff
@@ -20,34 +20,29 @@ export const DynamicEntry = () => {
     let component: IComponent;
     let prev = NULL_CHILD_STUB;
 
-    destroy$.pipe(
-        take(1)
-    ).subscribe(() => {
+    destroy$.pipe(take(1)).subscribe(() => {
         if (component) {
             component.destroy$.next(void 0);
         }
     });
 
-    update$.pipe(
-        takeUntil(destroy$),
-    )
-    .subscribe(curr => {
+    update$.pipe(takeUntil(destroy$)).subscribe((curr) => {
         // create a new component if:
         // - there were no prev value
         // - different data types of children
         // - different types of elements (leaf / array / static / observable / fn)
         // - keys mismatch
         // - xmlns mismatch
         // - number of children mismatch
-        if (prev === NULL_CHILD_STUB
-            || getType(prev) != getType(curr)
-            || (isElement(curr) && isElement(prev) && (
-                   prev.type !== curr.type
-                || !Object.is(prev.props.key, curr.props.key)
-                || !Object.is(prev.props.xmlns, curr.props.xmlns)
-                || prev.props.children.length !== curr.props.children.length
-                )
-            )
+        if (
+            prev === NULL_CHILD_STUB ||
+            getType(prev) != getType(curr) ||
+            (isElement(curr) &&
+                isElement(prev) &&
+                (prev.type !== curr.type ||
+                    !Object.is(prev.props.key, curr.props.key) ||
+                    !Object.is(prev.props.xmlns, curr.props.xmlns) ||
+                    prev.props.children.length !== curr.props.children.length))
```

Comment:
This reformatting makes me cry, but I'm ok to sacrifice it for greater good (tm) 🙂